CHANGELOG.md

```diff
@@ -14,11 +14,8 @@ the code was deployed.
 ### Changed
 
 - Updated to Device OS 3.3.1 (CU-3aru0mb).
-<<<<<<< HEAD
 - Replaced locationid with client display name in Sensor disconnection initial and reminder messages (CU-2q39wpk).
-=======
 - Sensor heartbeat API call body key changed from 'doorLastHeartbeat' to 'doorLastMessage' (CU-34atvnc).
->>>>>>> 1e43212 (Change the door last seen behaviour)
```

Comment:
Oops, somehow these got in during the last merge conflict. We'll fix that here

Reply:
oop thanks for the catch!

Reply:
Haha, yup! Funny.